privacypolicy.md

```diff
@@ -12,7 +12,7 @@ Giant Swarm collects personal data only when you provide it to us, through regis
 
 In order for the services of Giant Swarm to function, Giant Swarm needs to collect and process certain elements of your personal data in the course of a registration process. When you register, Giant Swarm asks for your name, mailing address, e-mail address and telephone number. If you use any of Giant Swarm’s paid services, Giant Swarm also collects your billing details such as your credit card or bank details for billing and payment purposes.
 
-The personal data Giant Swarm collects will only be used for the purpose of supplying you with the requested services or for other purposes for which you have given your consent, unless Giant Swarm is legally required to do so. Giant Swarm uses credit card or bank details information solely to check the financial qualifications and to collect payment for the services of Giant Swarm. Giant Swarm also uses your e-mail address in order to offer you similar services of our own. You may object to this use of your e-mail address at any time without incurring any costs simply by clicking on the link at the end of our e-mails or send your objection to the following e-mail address [support@giantswarm.io](mailto:support@giantswarm.io).
+The personal data Giant Swarm collects will only be used for the purpose of supplying you with the requested services or for other purposes for which you have given your consent, unless Giant Swarm is legally required to do so. Giant Swarm uses credit card or bank details information solely to check the financial qualifications and to collect payment for the services of Giant Swarm. Credit Card information is never stored or even touched by Giant Swarm but handled directly by a qualified third party that handles the billing process. Giant Swarm also uses your e-mail address in order to offer you similar services of our own. You may object to this use of your e-mail address at any time without incurring any costs simply by clicking on the link at the end of our e-mails or send your objection to the following e-mail address [support@giantswarm.io](mailto:support@giantswarm.io).
```

Comment:
I suggest to remove `or even touched`, since it's ambigious and not meaningful IMHO.